crowdsourcer/templates/crowdsourcer/authority_questions.html

```diff
@@ -11,6 +11,27 @@ <h1 class="mb-4 mb-md-5">
         {% endif %}
         {{section_title}}
     </h1>
+    <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-light">
+      <div class="container-fluid">
+        <div class="collapse navbar-collapse" id="navbarNavDropdown">
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+              Skip to Q
+            </a>
+            <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+            {% for q_form in form %}
+            <li><a class="dropdown-item" href="#q{{q_form.question_obj.number}}{% if q_form.question_obj.number_part %}{{ q_form.question_obj.number_part}}{% endif %}">Q{{q_form.question_obj.number}}{% if q_form.question_obj.number_part %}{{ q_form.question_obj.number_part}}{% endif %}</a></li>
```

Comment:
Do you think, rather than repeating `{{q_form.question_obj.number}}{% if q_form.question_obj.number_part %}{{ q_form.question_obj.number_part}}{% endif %}` everywhere, we should just have a `{{q_form.question_obj.slug}}` attribute/method on the Queston model, that abstracts all the stuff away for us?